### PR TITLE
Fill in details for CVE-2021-25741

### PR DESCRIPTION
**Pull request by cjcullen**

### CODE REVIEW

**Comment by cjcullen** [comment]
Validated locally:
```
$ ./validate-k8s.sh 2021/25xxx/CVE-2021-25741.json 
Checking 2021/25xxx/CVE-2021-25741.json ...
SUCCESS
```

**Review by tallclair — APPROVED** [review]
/lgtm

File: 2021/25xxx/CVE-2021-25741.json [hunk 1]
```diff
@@ -1,18 +1,111 @@
...
+            "version": "3.1"
+        }
+    },
+    "problemtype": {
+        "problemtype_data": [
+            {
+                "description": [
+                    {
+                        "lang": "eng",
+                        "value": "CWE-20: Improper Input Validation"
```
Comment (tallclair) [inline 2021/25xxx/CVE-2021-25741.json:87]: I already merged this, but would this be a better choice here? `CWE-59 | Improper Link Resolution Before File Access ('Link Following')` (copied from https://nvd.nist.gov/vuln/detail/CVE-2017-1002101)

Comment (tallclair) [reply 2021/25xxx/CVE-2021-25741.json:87]: It looks like this supports a list. https://cwe.mitre.org/data/definitions/362.html#:~:text=A%20race%20condition%20exists%20when,are%20not%2C%20this%20violates%20atomicity (race condition) is another option.

**Pull request merged**